patches/chromium/use_electron_resources_not_chrome.patch

```diff
@@ -0,0 +1,32 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <sattard@slack-corp.com>
+Date: Wed, 23 Oct 2019 14:17:18 -0700
+Subject: use electron resources not chrome
+
```

Comment:
Can you explain the why behind this patch and if/when we can potentially remove it? (for posterity)

Reply:
Removed this patch